Gems/LmbrCentral/Code/Mocks/LmbrCentral/Shape/MockShapes.h

```diff
@@ -56,5 +56,73 @@ namespace UnitTest
         MOCK_METHOD1(GenerateRandomPointInside, AZ::Vector3(AZ::RandomDistributionType randomDistribution));
         MOCK_METHOD3(IntersectRay, bool(const AZ::Vector3& src, const AZ::Vector3& dir, float& distance));
     };
+
+    class MockShape : public LmbrCentral::ShapeComponentRequestsBus::Handler
+    {
+    public:
+        AZ::Entity m_entity;
+        mutable int m_count = 0;
```

Comment:
it doesn't seem necessary for `m_count` to be mutable, I haven't seen it modified in a `const` environment.

Reply:
This is just code moved from vegetation_mocks, although I forgot to delete the old version (done now)/ I've removed mutable as well.